l10nzh.properties

```diff
@@ -2,13 +2,19 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=
-publicVersions=8.9,9.0,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10.0
+publicVersions=8.9,9.0,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10.0,10.1
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+10.1.description=Support SonarQube 10.1
+10.1.sqVersions=[10.1,LATEST]
+10.1.date=2023-07-07
+10.1.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.1
+10.1.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.1/sonar-l10n-zh-plugin-10.1.jar
+
 10.0.description=Support SonarQube 10.0
-10.0.sqVersions=[10.0,LATEST]
+10.0.sqVersions=[10.0,10.1.*]
```

Comment:
Shoot. I was forgetting/overlooking that you keep all of your versions public. To be honest, it's really not designed (or perhaps I should say "anticipated") that there would be a plugin version per SQ version. - Which is not to imply that I think you should change how you do this. 
AND unfortunately, public versions can't have overlapping compatibility, so this has to change back to the way you had it in the first place. Just to be explicit:
YOU WERE RIGHT. I WAS WRONG. :-D

And I'm not going to make you jump through the hoop. I'll approve this and then change it back myself. 
I'm just not sure whether users on 10.1 who are using your plugin version 10.0 will be prompted to upgrade. :-/